Binary_search/Stress_test.py

```diff
@@ -0,0 +1,38 @@
+import random
+from Binary_search import Bin_search as binkS
+from Simple_search import Simple_search as simS
+
+n = 20
+K = 0
+N = [0]
+
+# Генерируем случайный набор массивов
+random.randint(0,3)
+
+for i in range(10000):
+
+    N = [0]
+    for j in range(1,n):
+        N += [N[j-1]+random.randint(0,3)]

```

Comment:
That's a very strange implementation. Maybe, we can generate a random array and sort it with python `sorted` function? :)